test/runtests.jl

```diff
@@ -69,8 +70,8 @@ admit[:rank] = pool(admit[:rank])
         @test isapprox(aicc(gm2), 470.7312329339146)
         @test isapprox(bic(gm2), 494.4662797585473)
         @test isapprox(coef(gm2),
-            [-3.9899786606380734, 0.0022644256521549043, 0.8040374535155766,
-            -0.6754428594116577, -1.3402038117481079,-1.5514636444657492])
+            [-5.3301824723861895, 0.002264425652154903, 0.8040374535155792,
```

Comment:
I've seen your comment about why coefficients change, but that's still not clear to me. The order of levels should be the same as with PDAs AFAIK. I guess that's rather a change in StatsModels? Anyway, calling `levels!` should be enough to get the same behavior as before.

Reply:
The changes are with respect to the contrasts chosen for the levels of `admit[:rank]`, which is read as an integer then converted to categorical.  I think the `pool` conversion from a vector of integers to a PooledDataArray just uses the unique levels without sorting them whereas the `categorical` conversion does sort.  You would know better than I.

In any case, the contrasts chosen here are the sensible ones.  I would prefer not to enforce bug-for-bug compatibility with the last version.